styles.css

```diff
@@ -0,0 +1,70 @@
+.nav-bar {
+  display: flex;
+  align-items: baseline;
+  gap: 10px;
+  padding-top: 20px;
+  font-style: normal;
+  justify-content: flex-end;
+  color: blue;
+}
+
+a {
+  flex-grow: 20px;
+  font-style: normal;
+  margin-right: auto;
+  margin-left: 2rem;
+}
+
+p {
+  color: rgb(147, 147, 241);
+  width: 560px;
+}
+
+body {
+  background-color: white;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+  font-style: normal;
+  background-image: url(./background.png);
+  background-size: contain;
+}
+
+.headline-container {
+  display: flex;
+  align-content: flex-start;
```

Comment:
- [ ] As requested by the previous reviewer, Make sure that your design fits correctly the original design from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
The font, color, style, margin, and with of each element should be the same as on the Figma template.
- [ ] Also, you have to add a background image to your headline section as you can see on the Figma template. here below is a background image you can use:
![header-shapes mobile](https://user-images.githubusercontent.com/71874425/164762736-9b9fbece-f97b-4a05-a318-c4a7bfb89213.png)
- [ ] Make sure that the size(width) of your `p` tag with class="items item3" is the same as the design from the Figma template.
- Kindly, implement all the requested changes, ensure that your design is similar to the Figma template, and then request a re-review.